doc/developer-guide/documentation/RPC.en.rst

```diff
@@ -0,0 +1,397 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. _developer-doc-rpc:
+
+RPC
+*************
+
+========
+Overview
+========
+
+To communicate between :program:`traffic_manager` and :program:`traffic_server` there is an RPC mechanism in
+``mgmt/``. This is a simple serialization style RPC which runs over sockets.
+
+=======================================================
```

Comment:
Headlines need to be at least as long as the text. Because you added `:program:` this doesn't fit anymore. I recommend, though, taking that out and having just `Runtime Structure` as the heading or `Command line runtime structure`.

Reply:
renamed to `Runtime Structure`